src/main/java/com/javamas/araignee/mq/MessageQueueManager.java

```diff
@@ -0,0 +1,96 @@
+package com.javamas.araignee.mq;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import com.rabbitmq.client.QueueingConsumer;
+import com.rabbitmq.client.QueueingConsumer.Delivery;
+
+/**
+ * RabbitMQとの接続を保持するクラス.
+ * @author nayuta
+ */
+public class MessageQueueManager {
+
+	private ConnectionFactory factory = null;
+	private Connection connection = null;
```

Comment:
\[Checkstyle\] WARNING: 'member def modifier' have incorrect indentation level 8, expected level should be 2\.